ccmlib/node.py

```diff
@@ -1006,11 +1006,13 @@ def clear(self, clear_all=False, only_data=False, saved_caches=False):
             if only_data:
                 for dir in os.listdir(full_dir):
                     keyspace_dir = os.path.join(full_dir, dir)
-                    if os.path.isdir(keyspace_dir) and dir != "system":
+                    if os.path.isdir(keyspace_dir) and not dir.startswith("system"):
                         for f in os.listdir(keyspace_dir):
```

Comment:
I think this is the core of the fix, but the description doesn't really explain it

Also reference to that dtest is broken cause of this would help with understand the reason for the chage

Reply:
Actually, for me the other change was more important as it did not remove sstable files. My tests were passing but repair was not triggered.